pkgs/applications/science/math/geogebra/geogebra6.nix

```diff
@@ -18,9 +18,8 @@ let
     src = fetchurl {
       urls = [
         "https://download.geogebra.org/installers/6.0/GeoGebra-Linux64-Portable-${version}.zip"
-        "https://web.archive.org/web/20210604132845/https://download.geogebra.org/installers/6.0/GeoGebra-Linux64-Portable-${version}.zip"
```

Comment:
Here is updated link to webarchive:
`"https://web.archive.org/web/20211123222708/https://download.geogebra.org/installers/6.0/GeoGebra-Linux64-Portable-${version}.zip"`
Otherwise, once upon a time upstream download link will not work (they keep only last 4 releases).

Reply:
ahh thank you for doing that. I had a little bit of problems with the archive website recently